app/src/androidTest/java/mozilla/lockbox/OnboardingTest.kt

```diff
@@ -49,8 +56,11 @@ open class OnboardingTest {
         navigator.gotoAutofillOnboarding()
         autofillOnboardingScreen { tapSkip() }
         navigator.checkAtOnboardingConfirmation()
+        onboardingConfirmationScreen { clickFinish() }
+        navigator.disconnectAccount()
```

Comment:
do you need to have this both in the test and in the `BeforeTest`?

Reply:
you are right, we don't need this here :)